plugins/ltac2/tac2ffi.mli

```diff
@@ -182,6 +186,7 @@ val val_binder : (Name.t EConstr.binder_annot * types) Val.tag
 val val_free : Id.Set.t Val.tag
 val val_uint63 : Uint63.t Val.tag
 val val_float : Float64.t Val.tag
+val val_pstring : Pstring.t Val.tag
```

Comment:
I think we don't need to expose this. The other `val_*` have not been better than exposing to/of/repr functions AFAICT

Reply:
That was only for consistency, shall I remove this?

Reply:
yes